grakn-graql/src/main/java/ai/grakn/graql/internal/analytics/DegreeDistributionMapReduce.java

```diff
@@ -47,14 +47,16 @@
     public DegreeDistributionMapReduce() {
     }
 
-    public DegreeDistributionMapReduce(Set<TypeId> selectedTypeIds) {
+    public DegreeDistributionMapReduce(Set<TypeId> selectedTypeIds, String degreeLabel) {
```

Comment:
Degree is a property on the vertex. So I guess I could change the variable name to sth like degreePropertyKey?